app/frontend/src/features/auth/email/ChangeEmailPage.tsx

```diff
@@ -55,7 +63,9 @@ export default function ChangeEmailPage() {
 
   return (
     <>
-      <PageTitle>Change email</PageTitle>
+      <PageTitle>{CHANGE_NAME_GENDER}</PageTitle>
+      <Typography variant="body1">{CONTACT}</Typography>
+      <PageTitle>{CHANGE_PASSWORD}</PageTitle>
```

Comment:
Password change is a different page. Maybe there needs to be a new page "Account settings" And put all three forms (name/gender contact info, email change, password page) on?

Reply:
That's why I tried to make this page for

Reply:
Oops :D